src/main/java/com/denizenscript/ddiscordbot/objects/DiscordGroupTag.java

```diff
@@ -189,6 +189,44 @@ public static void register() {
             return list;
         });
 
+        // <--[tag]
+        // @attribute <DiscordGroupTag.boosters>
+        // @returns ListTag(DiscordUserTag)
+        // @plugin dDiscordBot
+        // @description
+        // Returns a list of all users in the group that currently boosts the group.
+        // -->
+        tagProcessor.registerTag(ListTag.class, "boosters", (attribute, object) -> {
+            ListTag list = new ListTag();
+            for (Member member : object.getGuild().getBoosters()) {
+                list.addObject(new DiscordUserTag(object.bot, member.getUser()));
+            }
+            return list;
+        });
+
+        // <--[tag]
+        // @attribute <DiscordGroupTag.boosts_count>
+        // @returns ElementTag(Number)
+        // @plugin dDiscordBot
+        // @description
+        // Returns the amount of boosts the group currently has.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "boosts_count", (attribute, object) -> {
+            return new ElementTag(object.getGuild().getBoostCount());
+        });
+
+        // <--[tag]
+        // @attribute <DiscordGroupTag.tier>
+        // @returns ElementTag(Number)
+        // @plugin dDiscordBot
+        // @description
+        // Returns the tier of the group currently set by its boosts.

```

Comment:
Maybe just `Returns the group's current boost tier, see <@link> for all possible tiers.`?